### DISCUSSION TITLE
Modal link inside clickable table row not working properly

### DISCUSSION THREAD

**Discussion by DonCamillo11** [livewire/livewire#8029] [1. Help]
I have a clickable table row in my table
`<x-table.tr wire:key="{{ $job->id }}" wire:click.prevent="show('{{ $job->id }}')" class="cursor-pointer">`
inside this tr there is a button where I want to open a delete confirmation modal
```
<a href="#" wire:click.stop="confirmUserDeletion('{{$job->id}}')">Delete</a>
@include('livewire.dialog.delete')
```
But, every time I open the modal and click the cancel or the confirmation button the click event from the tr is also triggered and executed after that. I already added the wire:click.stop to stop the propagation, but this doesn’t help. Has anyone an idea of how to fix this?

**Comment by joshhanley** [comment]
@DonCamillo11 any chance you could create a `https://wirebox.app` that demonstrates the issue so we can have a look?
